recipes-kernel/linux/5.10/patches/xsa-155-qsb-023-xen-netfront-do-not-use-data-already-exposed-to-back.patch

```diff
@@ -57,3 +57,54 @@ PATCHES
  
  	if (skb->ip_summed == CHECKSUM_PARTIAL)
  		/* local packet? */
+@@ -907,9 +908,8 @@ static int xennet_get_responses(struct n
+ 				struct sk_buff_head *list,
+ 				bool *need_xdp_flush)
+ {
+-	struct xen_netif_rx_response *rx = &rinfo->rx;
+ 	struct xen_netif_rx_response rx = rinfo->rx;
+-	int max = XEN_NETIF_NR_SLOTS_MIN + (rx->status <= RX_COPY_THRESHOLD);
++	int max = XEN_NETIF_NR_SLOTS_MIN + (rx.status <= RX_COPY_THRESHOLD);
```

Comment:
This patch has a fixup of the previous patch, xsa-155-qsb-023-xen-netfront-copy-response-out-of-shared-buffer-befo.patch .  It should be moved there.